src/sass/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,3 @@
+.slds-breadcrumb__item {
```

Comment:
Careful about overriding slds classes – better to add a new class?

Reply:
In MD we're doing this same override, but only within a custom class: https://github.com/SFDO-Tooling/MetaDeploy/blob/master/src/sass/components/_header.scss#L16-L32

Maybe we should take a similar approach here, nesting this within `.ms-breadcrumb` or something like that?